DomainContent/Zaru/applause-app/loadApplauseApp.js

```diff
@@ -0,0 +1,22 @@
+// loadApplauseApp.js
+// 
+// Loads the applause app on to a user's device while they are in the vicinity
+// Created by Liv Erickson 6/18/2018 
+// 
+// Copyright 2018 High Fidelity Inc.
+//
+// Distributed under the Apache License, Version 2.0.
+// See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+(function(){
+    var HAS_APPLAUSE_APP_SETTING = 'io.highfidelity.applauseEnabled.appPresent';
+    var ApplauseItem = function() {};
+    ApplauseItem.prototype = {
+        preload: function(entityID) {
+            if (!Settings.getValue(HAS_APPLAUSE_APP_SETTING)) {
+                Script.require('./applause-app.js');
+            }
+        }
+    };
+    return new ApplauseItem();
```

Comment:
Should this remove the app after they leave the zone or go to another domain?  I think I usually do this with ScriptDiscoveryService.stop(appUrl)

Reply:
The script is only present when they're in the domain as is - I believe the entity script unloads the required modules it pulls in when it's unloaded, which triggers the ScriptEnding event on the `applause-app.js` and removes it.